db/schema.rb

```diff
@@ -398,6 +399,7 @@
   add_index "users", ["display_name"], name: "users_display_name_trgm_index", using: :gist, opclass: {"display_name"=>:gist_trgm_ops}
```

Comment:
Searching for users. The collaborator editor searches by display name and I assume the @ mentions could too. Similar to the interface for mentioning people on  Github/Twitter/Facebook.